src/core/analytics/analyticsreporter.js

```diff
@@ -99,7 +99,14 @@ export default class AnalyticsReporter {
       throw new AnswersAnalyticsError('Tried to send invalid analytics event', event);
     }
 
-    event.addOptions(this._globalOptions);
+    if (includeQueryId) {
+      event.addOptions(this._globalOptions);
```

Comment:
is this needed for when a user specifies a queryId in ANSWERS.init's config.analyticsOptions?

Reply:
By default we send the last queryId with every analytics event. For voice search, we shouldn't send one so I added this so that we could send the analytics events without one

Reply:
For fun I was wondering when this was added https://github.com/yext/answers-search-ui/pull/196/files
I was a little worried this would encourage people to add more flags in the future, but eh I can't think of a better way than what you have

Reply:
Would a `reportWithoutQueryId` function work better than a flag?

Reply:
yeah that works!

Reply:
I actually just updated the function to have an optional `options` object as the second parameter which is clearer than the lone boolean flag and it more easily lets us add additional functionality. What do you think of that approach?